Page builder editor (#107)
ezsystems/user-documentation: docs/site_organization/working_with_page.md

On the **Design** tab, in the **View** field, change the layout to "Products" and submit your changes.

## Configure blocks display

As an administrator, you can define which page blocks elements are available for an editor in the page edit mode.

> Go to the Content Type you want to edit. In the **Field definitions** section, click **Landing page** tab.

Reviewer: ```suggestion
Go to the Content Type that you want to edit. 
In the **Field definitions** section, click the **Landing page** tab.
```

Expand the **Select blocks** section, and select page blocks.

![Page blocks](img/select_page_blocks.png "Select page blocks")

> To confirm changes, click **Save**.

Reviewer: ```suggestion
Click **Save** to apply the changes.
```

Now, only selected page blocks are available in the edit mode.

![Elements menu](img/page_blocks_toolbar.png "Elements menu")

!!! caution

>     Deselecting blocks will hide any related blocks inlcluded in the Landing page. To publish the Page,

Reviewer: ```suggestion
    When you deselect blocks, any related blocks that are included in the Landing page hide as well. 
    To publish the Page,
```

>     a user will have to delete these blocks from the Landing Page.

Reviewer: ```suggestion
    a user has to delete these blocks from the Landing Page.
```

!!! caution "Add new blocks"

>     If a [developer creates a new block](https://doc.ibexa.co/en/latest/extending/extending_page/) and you want to make it available in the editor mode, you must select it manually in the **Select page blocks** section.

Reviewer: ```suggestion
    If a [developer creates a new block](https://doc.ibexa.co/en/latest/extending/extending_page/) and you want to make it available 
    in the editor mode, you must select it manually in the **Select page blocks** section.
```

## Configure preferred edit mode

> As an administrator, you can set the edit mode which will be launched when an editor starts editing the Page.

Reviewer: ```suggestion
As an administrator, you can set the edit mode that is launched when an editor starts editing the Page.
```

> Go to the Content Type you want to edit. In the **Field definitions** section, click the **Landing page** tab.

Reviewer: ```suggestion
Go to the Content Type that you want to edit. 
In the **Field definitions** section, click the **Landing page** tab.
```

Expand the **Select Editor launch mode** section, and select from the available options.

![Editor launch mode](img/select_editor_mode.png "Select Editor launch mode")

## Configure available page layouts

As an administrator, you can configure which page layouts are available for an editor.
On the **Admin** tab, click **Content Types**.

> Go to the Content Type you want to edit. In the **Field definitions** section, expand **Landing page** tab.

Reviewer: ```suggestion
Go to the Content Type that you want to edit. 
In the **Field definitions** section, expand the **Landing page** tab.
```

> Expand the **Select layouts** section, and define which layout will be available for this Page.
> To confirm changes, click **Save**.

Reviewer: ```suggestion
Expand the **Select layouts** section, and define layouts that are available for this Page.
Click **Save** to apply the changes.
```

> If you deselect a layout that is currently used, the user will have to change the layout to be able to edit the Page.

Reviewer: ```suggestion
If you deselect a layout that is currently used, the user has to change the layout to be able to edit the Page.
```
"Change the layout" - is it "select a different layout"?
